waku/v2/waku_archive/driver/builder.nim

```diff
@@ -69,6 +71,18 @@ proc new*(T: type ArchiveDriver,
 
     return ok(res.get())
 
+  of "postgres":
+    const MaxNumConns = 5 #TODO: we may need to set that from app args (maybe?)
+    let res = PostgresDriver.new(url, MaxNumConns)
+    if res.isErr():
+      return err("failed to init postgres archive driver: " & res.error)
+
+    let driver = res.get()
+    # The table should exist beforehand.
+    discard driver.createMessageTable()
```

Comment:
the table should exist but perhaps its still valuable to check isErr?

Reply:
I thought that was not important but now that you mention I'd check it :)
Thanks !